src/PEAR.php

```diff
@@ -232,7 +232,7 @@ public static function __callStatic($method, $arguments)
             );
         }
         return call_user_func_array(
-            array(self::class, '_' . $method),
+            array(get_class(), '_' . $method),
```

Comment:
This emits php warning on PHP 8.3:
```
PHP Deprecated:  Calling get_class() without arguments is deprecated in /home/runner/work/roundcubemail/roundcubemail/vendor/pear/pear-core-minimal/src/PEAR.php on line 235
```

Reply:
@mvorisek , I understand... my point is that such fixes need to _originate_ in the `pear-core` repo, not here in `pear-core-minimal`... we have a process in place to get changes from `-core` down to `-minimal`, so that their files stay in sync.

The best path forward for **PHP 8.3** compatibility is to submit **PR**s to `pear-core`.  As those fixes are merged, I'll get them ported into here.